sql/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -313,4 +313,49 @@ object Connect {
       .internal()
       .booleanConf
       .createWithDefault(true)
+
+  val CONNECT_SESSION_ML_CACHE_TIMEOUT =
+    buildConf("spark.connect.session.mlCache.timeout")
```

Comment:
I am not sure if we can do this. This is a non-recoverable situation for a client. The client itself should tell us they are done with a model. I would suggest instead that we either fail and give them a list of models they can evict, or that we store old models on disk.